README.md

```diff
@@ -42,6 +42,10 @@ sh update_modules.sh
     &*/:`<>?\|"
     ```
 - Images must be `.png` format
-- Images should be no wider than 512px
+- Image resolution guidelines:
+  - Images with native width larger than 512px should be scaled down to 512px wide
+  - Images with native width of 512px should be added as-is
+  - Images with native width 256px and 512px should be scaled down to 256px wide
```

Comment:
Should this be "Images with native width **between** 256px and 512px..."?

Reply:
You're right. Fixed, thanks!